net-libs/zeromq/zeromq-4.3.3-r1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools
+
+DESCRIPTION="A brokerless kernel"
+HOMEPAGE="http://www.zeromq.org/"
+SRC_URI="https://github.com/zeromq/libzmq/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0/5"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~x64-macos ~x86-macos"
+IUSE="doc drafts pgm +sodium static-libs test unwind elibc_Darwin"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	!elibc_Darwin? ( unwind? ( sys-libs/libunwind ) )
+	sodium? ( dev-libs/libsodium:= )
+	pgm? ( >=net-libs/openpgm-5.2.122:= )"
+DEPEND="${RDEPEND}
+	!elibc_Darwin? ( sys-apps/util-linux )
+	doc? (
```

Comment:
The doc and pkgconfig is probably BDEPEND?

Reply:
Probably. I'd leave that to the package maintainer.